src/cpp/src/llm_pipeline.cpp

```diff
@@ -59,23 +59,31 @@ class StatefulLLMPipeline final : public LLMPipelineImplBase {
         const ov::genai::Tokenizer& tokenizer,
         const std::string& device,
         const ov::AnyMap& plugin_config
-    ) : LLMPipelineImplBase(tokenizer, utils::from_config_json_if_exists(models_path))
-    {
+    ) : StatefulLLMPipeline{
+            ov::genai::utils::read_model_with_config(models_path, plugin_config),
```

Comment:
why `read_model_with_config` is not used in all places? 

does it read model with enable_mmap flag explicitly set?

Reply:
Both creations with models_path go through this ctor, so it's the only place where we need `read_model_with_config`.

It ensures that MMAP config is passed to the core.